README.md

```diff
@@ -178,16 +180,16 @@ Here is a real-world example including files and folders to check, as well as so
 
     <!-- Do not report on function length for tests. -->
     <rule ref="Syde.Functions.FunctionLength">
-        <exclude-pattern>*/tests/cases/*</exclude-pattern>
+        <exclude-pattern>*/tests/*</exclude-pattern>
     </rule>
 
     <!-- PSR-4 namespace configuration. -->
     <rule ref="SlevomatCodingStandard.Files.TypeNameMatchesFileName">
         <properties>
             <property name="rootNamespaces" type="array">
                 <element key="src" value="MyCompany\MyProject" />
-                <element key="tests/cases" value="MyCompany\MyProject\Tests" />
                 <element key="tests/src" value="MyCompany\MyProject\Tests" />
+                <element key="tests/unit" value="MyCompany\MyProject\Tests" />
```

Comment:
why has `tests/src` the same namespace as `tests/unit`? Should we maybe add to the second on `\Unit` to the namespace? 👀

Reply:
This is example code only. I wanted to illustrate how to migrate from the Inpsyde sniff, which supports pipe-separated paths for a namespace, to the Slevomat sniff, which does not support multi-path configuration.